contracts/Comet.sol

```diff
@@ -163,8 +173,6 @@ contract Comet is CometMainInterface {
         if (config.assetConfigs.length > MAX_ASSETS) revert TooManyAssets();
         if (config.baseMinForRewards == 0) revert BadMinimum();
         if (AggregatorV3Interface(config.baseTokenPriceFeed).decimals() != PRICE_FEED_DECIMALS) revert BadDecimals();
-        if (config.reserveRate > FACTOR_SCALE) revert BadReserveRate();
-        if (config.kink > FACTOR_SCALE) revert BadKink();
```

Comment:
I removed this check because we theoretically have an IR model with a kink above 100%.